### PR TITLE
[Components with more than 1 service] LPS-180879 Refactor ScreenNavigationCategory components from USM modules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://issues.liferay.com/browse/LPS-180879

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:usm

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/frontend/taglib/servlet/taglib/ServerPortalPropertiesScreenNavigationCategory.java [hunk 1]
```diff
@@ -15,48 +15,33 @@
...
 
 	@Override
 	public String getCategoryKey() {
 		return ServerAdminNavigationEntryConstants.
 			CATEGORY_KEY_PORTAL_PROPERTIES;
 	}
 
-	@Override
-	public String getEntryKey() {
-		return ServerAdminNavigationEntryConstants.ENTRY_KEY_PORTAL_PROPERTIES;
```
Comment (dantewang) [inline modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/frontend/taglib/servlet/taglib/ServerPortalPropertiesScreenNavigationCategory.java:54]: @MrJohn1911 There's still a usage to this constant. To make it safe, if we choose to delegate, we'd better remove the `ENTRY_KEY_` constant, replacing all its usages with the `CATEGORY_KEY_` one.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/frontend/taglib/servlet/taglib/ServerPortalPropertiesScreenNavigationCategory.java (hunk 1 above)
Comment (MrJohn1911) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/frontend/taglib/servlet/taglib/ServerPortalPropertiesScreenNavigationCategory.java:54]: @dantewang So, you want me to do this in this specific case or to all `ENTRY_KEY` constants that I stopped using in my pull requests?

Comment (dantewang) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/frontend/taglib/servlet/taglib/ServerPortalPropertiesScreenNavigationCategory.java:54]: @MrJohn1911 OK, let's only do the delegate (`getEntryKey() -> getCategoryKey()`) when both methods are using the same constant, not when the constants have the same content/value.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:usm

**Comment by dantewang** [comment]
ci:forward

cc @shuyangzhou @tinatian

**Pull request closed without merging**